app/views/taxYearOverview.scala.html

```diff
@@ -64,7 +64,8 @@ <h1 class="heading-xlarge">
 
             <table id="income-deductions-table">
                 <tr>
-                    <td>@messages("tax-year-overview.income")</td>
+                    <td id="income-row-title"><a href=@controllers.routes.IncomeSummaryController.showIncomeSummary(taxYear).url>
```

Comment:
The link needs to be conditional based on the feature switch, as it stands if the feature switch is disabled the link still appears and routes to a Not Found page which is a poor user journey.